utility/src/main/AndroidManifest.xml

```diff
@@ -1 +1,5 @@
-<manifest package="org.oppia.util"/>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="org.oppia.util">
+  <uses-sdk android:minSdkVersion="19"
+    android:targetSdkVersion="29" />
```

Comment:
Does this actually need to be 28 per the issue you mentioned in #1495? Ditto for the test version of the manifest.

Reply:
yes, will change now!